### DISCUSSION TITLE
View coverage for upcoming ranked tracks

### DISCUSSION THREAD

**Discussion by rickybloomfield** [MKT-Playbook/Hub#46] [Feature requests]
I'm not sure if this app is still under active development, but I use it all the time as a paid user! 🙏

Now that we know the ranked tracks at least 3 tours in advance, it would be great to have an additional "tool" to show us this coverage so we can plan our upgrades accordingly. Studfit used to create an amazing graphic showing exactly this (using the Larry98 sheets), but he has stopped his updates. Thanks!

**Comment by TitusPosiadly** [comment]
Hey, thanks for the suggestion.

The app is currently more in maintenance mode, where I try to keep all the data up to date for the foreseeable future. I probably won't be adding any more big features, at least for now, but maybe I will have the drive to implement something new sometime next year. Who knows.

That being said adding a coverage checker for all ranked tracks should be quite easy. I could just add a new filter option for the CoverageCrafter. And as far as I know, we already know all ranked courses for the entire year (and all years to come because the last year will be on repeat from the Battle Tour (2023) onwards). So I could add an option like "Limit to ranked courses" that will only consider the 156 ranked tracks (26 tours per year * 2 ranked weeks per tour * 3 courses per week)(will probably have some duplicated) when calculating the coverage. Would that be close enough to what you were thinking of?

If you have any more info on this (like the old Studfit graphic) let me know and I will try to get a small update ready soon.

**Reply by rickybloomfield** [reply]
Thanks! Yes, that's pretty much what Studfit did. You can see his last Reddit post [here](https://www.reddit.com/r/MarioKartTour/comments/17d0ph0/ranked_usage_for_rest_of_year_5_from_halloween/). Screenshots 2-4 show how often each D/K/G is used in ranked over the next year (he filtered drivers to those with plus items and karts to those that are large, but I don't think that would be necessary in this case). Screenshots 6-8 are also helpful - these show the D/K/Gs used over the next 3 tours so you can prioritize upgrading things now that will be the most useful over the next 1 1/2 months.

I totally understand that the app is in maintenance mode, so no worries if you can't get to this. Assuming they keep repeating the tours from this year, though, this could be a very useful change for users. Thanks!

**Reply by robertybob** [reply]
I came here to ask for the same thing 😄 I didn’t realise the app was in maintenance mode when I purchased it, it’s understandable but still a shame to hear 😞

**Reply by TitusPosiadly** [reply]
Although the app is in maintenance mode, I do have a small update with at least some new features scheduled. I don't know if I will be able to rework the CoverageCrafter just yet, though. But I did add some useful ranked features to the card list and detailed page. The update will be released around the time the new iOS version is released to the public.

**Reply by TheBlacksmth** [reply]
Mario/Luigi Mechanics and the Titans need to be updated to include new favorites in the Sundae Tour. They don’t appear in the calculations for their new favorites.

**Reply by TitusPosiadly** [reply]
Yes, thanks for reaching out. It took a bit longer today to get the necessary data, but I got it now and added the missing data to the database. The change should be reflected in your app the next time you open it.
